web/src/components/CreateShortcutDialog.tsx

```diff
@@ -181,11 +187,18 @@ const MemoFilterInputer: React.FC<MemoFilterInputerProps> = (props: MemoFilterIn
 
   useEffect(() => {
     if (type === "DISPLAY_TIME") {
```

Comment:
```suggestion
    if (type === "DISPLAY_TIME" && filter.value.value === "") {
```

I think we only need to add this condition to check whether it is a `creation` or an `editing` state. WDYT?

Reply:
Hi thanks for your comment. Yes I guess this will work too :) I will adjust it and try to create a new pull request